cc3d/player5/Graphics/GraphicsFrameWidget.py

```diff
@@ -102,7 +104,15 @@ def __init__(self, parent=None, originatingWidget=None):
         self.renWin = self.qvtkWidget.GetRenderWindow()
         self.renWin.AddRenderer(self.gd.get_renderer())
 
-        self.metadata_fetcher_dict = {
+    @property
+    def metadata_fetcher_dict(self) -> typing.Dict[str, typing.Callable[[str, str], dict]]:
```

Comment:
Since this si a property maybe we can rename it to `metadata_dict` - if it does not make sense ignore the comment